initializers.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/go-coldbrew/log/loggers"
 	"github.com/go-coldbrew/log/loggers/gokit"
 	nrutil "github.com/go-coldbrew/tracing/newrelic"
+	protov1 "github.com/golang/protobuf/proto" //nolint:staticcheck
```

Comment:
Did we add a check in DomainEntities to not allow for lint ignores?
```
lint:
      disallow_comment_ignores: true
```

Reply:
no, we have not set `disallow_comment_ignores: true` anywhere in coldbrew